docs/section23/Rule23-10.md

```diff
@@ -0,0 +1,53 @@
+
+# Airside System - Rule 23-10  
+
+**Rule ID:** 23-10  
+**Rule Description:** System 5-8 and 11 - part load VAV fan power shall be modeled using either method 1 or 2 in Table G3.1.3.15. This rule will only validate data points from Method-1 Part-load Fan Power Data.  
```

Comment:
This effectively verifies both methods, because it is my understanding that the difference between the two methods is just in a way variation in input power is expressed, not in what it should be.

Reply:
I would add to the rule description "however, both methods are equivalent. When modeling inputs are based on Method 2, values should be converted to Method 1 when writing to RMD

Reply:
Rule description is updated.